.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] It will be nice to add the node-modules file to the .gitignore. As it can be added by accident while setting up linters

Reply:
node_modules was added